litho-core/src/main/java/com/facebook/litho/ComponentTree.java

```diff
@@ -2086,28 +2088,46 @@ public void release() {
               @Override
               public void run() {
                 final LayoutState layoutState = get();
+                if (layoutState == null) {
+                  // LayoutState is already released, ignore.
+                  return;
+                }
                 layoutState.releaseRef();
                 // get() calls acquireRef, so we need to release again.
                 layoutState.releaseRef();
               }
             });
       } else {
         final LayoutState layoutState = get();
```

Comment:
If you're already set isReleased on L2082, get() will always return null won't it?

Reply:
yeah great catch here. I created a `getInternal()` to get around this